src/server/pfs/fuse/server.go

```diff
@@ -1401,10 +1391,31 @@ func mountingState(m *MountStateMachine) StateFn {
 			Write:    m.Mode == "rw",
 		}
 		m.manager.root.branches[m.Name] = m.Branch
-	}()
+
+		// Get the latest non-alias commit on branch
+		// TODO: notebooks support
```

Comment:
What's the TODO here?

Reply:
Just a reminder to myself to update this code when I add projects support to the mount server. Here, it'll be passing the appropriate project in the `InspectBranch` rpc.